### DISCUSSION TITLE
can we integrate TypeOrm with neutralinojs?

### DISCUSSION THREAD

**Discussion by MeetChaudhari** [neutralinojs/neutralinojs#935] [Q&A]
can we integrate TypeOrm with neutralinojs?

**Comment by pegvin** [comment]
What is typeorm? And what are the benefits of integrating it in Neutralinojs?

**Reply by MeetChaudhari** [reply]
TypeORM is an Node based ORM, and integrating it will allow us to use database for storage rather than the storage api.
